### PR TITLE
Latest cards

### PR DESCRIPTION
**Pull request by ryanprice**
New card types
text changes
Feedback component added
Booster minting throttle added temporarily

### CODE REVIEW

File: src/components/BodyContent.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
           <p>
             <h2>ZOOM Token Liquidity Sale Event !</h2>
             <p><span class="text-danger">NOTE:</span> the ZOOM ERC20 token in a game utility token for the Zoombies NFT World, <router-link to="/help">please read and understand the mint and burn operations of this token</router-link> before purchase.<br />
-              <br/>Holding a sufficient ZOOM token balance for a card type will unlock FREE and reduced Shop NFT minting costs
+              <br/>Holding a sufficient ZOOM token balance for a card type will unlock FREE and reduced Shop NFT minting costs, and provide a burn option to increase chance of rarity when minting booster NFTs. Future utility will include ZOOM tokens as a form of health points.
```
Comment (jjjhill) [inline src/components/BodyContent.vue:23]:
```suggestion
              <br/>Holding a sufficient ZOOM token balance will unlock some FREE cards and reduce the minting costs of some Shop NFTs. ZOOM tokens can also be burned to increase the chance of pulling higher rarity cards when minting booster NFTs. Future utility will include ZOOM tokens as a form of health points.
```

File: src/components/BodyContent.vue [hunk 2]
```diff
@@ -165,6 +167,21 @@ export default {
       return parseInt(this.myPurchaseTotal/100000000000).toLocaleString();
     }
   },
+  mounted() {
+      this.$nextTick(function () {
+         if (this.CzxpInstance) {
+           this.updateSale();
+         }
+      });
```
Comment (jjjhill) [inline src/components/BodyContent.vue:175]: This `mounted` code block can be removed as it was replaced by the `watch` code block below

File: src/App.vue [hunk 3]
```diff
@@ -191,10 +191,10 @@ export default {
         web3.eth.net.getId(),
       ]);
 
-      if(networkId !== 1285){
```
Comment (mchigit) [inline src/App.vue:194]: OOC why is this part commented out? I'm asking cause I think this code block will configure us to connect to test net and I don't know if there is something I didn't consider

**Review by mchigit — APPROVED** [review]
image

Just a side note, should we re-design the tablet/mobile view to make the header items appear vertically instead of using a scroll? We have a lot of items on the nav and even in desktop it's beginnng to look a bit cluttered. 

Everything else LGTM

File: src/App.vue (hunk 3 above)
Comment (jjjhill) [reply src/App.vue:194]: i think i just commented it out for local dev, i'll uncomment

**Comment by jjjhill** [comment]
> image
> 
> Just a side note, should we re-design the tablet/mobile view to make the header items appear vertically instead of using a scroll? We have a lot of items on the nav and even in desktop it's beginnng to look a bit cluttered.
> 
> Everything else LGTM

agreed @ryanprice let us know when to prioritize this

**Pull request merged**